lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -422,8 +443,74 @@
                             "provisionVMAgent": true
                         }
                     }
+                },
+                "getOSProfile": {
+                    "parameters": [
+                        {
+                            "name": "computername",
+                            "type": "string"
+                        },
+                        {
+                            "name": "admin_username",
+                            "type": "string"
+                        },
+                        {
+                            "name": "password",
+                            "type": "string"
+                        },
+                        {
+                            "name": "linuxConfiguration",
+                            "type": "string"
+                        }
+                    ],
+                    "output": {
+                        "type": "object",
+                        "value": {
+                            "computername": "[parameters('computername')]",
+                            "adminUsername": "[parameters('admin_username')]",
+                            "adminPassword": "[parameters('admin_password')]",
+                            "linuxConfiguration": "[parameters('linuxConfiguration')]"
+                        }
+                    }
+                },
+                "getOSImage": {
+                    "parameters": [
+                        {
+                            "name": "node",
+                            "type": "object"
+                        }
+                    ],
+                    "output": {
+                        "type": "object",
+                        "value": {
+                            "name": "[concat(parameters('node')['name'], '-osDisk')]",
+                            "diffDiskSettings": "[if(equals(parameters('node')['disk_type'], 'Ephemeral'), json('{\"option\": \"local\", \"placement\": \"CacheDisk\"}'), json('null'))]",
+                            "managedDisk": "[if(not(equals(parameters('node')['disk_type'], 'Ephemeral')), json(concat('{\"storageAccountType\": \"',parameters('node')['disk_type'],'\"}')), json('null'))]",
+                            "caching": "[if(equals(parameters('node')['disk_type'], 'Ephemeral'), 'ReadOnly', 'ReadWrite')]",
+                            "createOption": "FromImage",
+                            "diskSizeGB": "[parameters('node')['osdisk_size_in_gb']]"
+                        }
+                    }
+                },
+                "getOSDisk": {
+                    "parameters": [
+                        {
+                            "name": "diskName",
+                            "type": "string"
+                        }
+                    ],
+                    "output": {
+                        "type": "object",
+                        "value": {
+                            "createOption": "Attach",
```

Comment:
Does the attach method support to run in parallel? My concern is the attached vhd supports only one vm, and it's dirty after used once.

Reply:
I confirmed that deployment works with two VMs (setting min_count to 2) works.
Is there another scenario that I should test?